Apps/WebClient/src/ClientApp/app/views/timeline.vue

```diff
@@ -297,6 +305,8 @@ export default class TimelineComponent extends Vue {
 
   @Ref("protectiveWordModal")
   readonly protectiveWordModal!: ProtectiveWordComponent;
+  @Ref("covidModal")
+  readonly covidModal!: CovidModalComponent;
```

Comment:
not sure we need this reference since its only used to set the showing to the component. Instead we can just pass it as a prop (I think is already passing it)

Reply:
Yes, the problem with showing the component is that the close is handled by the actual modal which makes this a two way binding for a "show", it just raises the complexity, + i'm following how the protective word modal works today!

Reply:
fair point